public/client/search.js

```diff
@@ -29,4 +46,42 @@ function request(bookId, pointCost, title){
             alert('Error occured while retrieving book')
         }
     })
+}
+
+function bookDetails(isbn, pointCost, userPoints){
+        $.ajax({
+            url: '/googleapi/isbn/' + isbn,
+            method: 'GET',
+            dataType: 'json',
+            success: function (res) {
+                console.log(res);
+                
+                var details = document.getElementById('details');
+                
+                details.innerHTML = `Title: ${res.title}
+                                    <br>
```

Comment:
Same with this. Please add a modal on the html page and then fill the modal with the response similar to how it's done throughout the application already.

Reply:
By this, all of the property labels and styling is done in html and you are going to simply fill the text or input values with the response values.

Reply:
I have applied this suggestion.